src/test/java/hudson/plugins/jira/JiraSite2Test.java

```diff
@@ -14,7 +14,7 @@
     @Test
     public void testClientInitialization() throws Exception {
         JiraSite site = new JiraSite(new URL("https://nonexistent.url"), null,
-                "user", "password",
+                (String)null,
```

Comment:
why cast to String?

Reply:
Emm, I found this change make the test not functional as expected, I'll fix it :(